lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -282,6 +285,49 @@ class ReportUIFeatures {
     this._dom.resetTemplates();
   }
 
+  /* eslint-env browser */
+  /**
+   * Jumps to respective audit on page load if hash is provided in URL.
+   * Collapsed content is opened automatically.
+   * Element is highlighted via CSS selector.
+   */
+  _jumpToAudit() {
+    const hash = window.location.hash;
+    if (!hash) {
+      return;
+    }
+    const audit = this._document.querySelector(hash);
+    if (!audit || !audit.classList.contains('lh-audit')) {
+      return;
+    }
+    if (audit.firstElementChild && audit.firstElementChild.tagName === 'DETAILS') {
+      audit.firstElementChild.setAttribute('open', 'true');
+    }
+    this._openClosestDetails(audit);
+  }
+
+  /* eslint-env browser*/
+  /**
+   * Open all closest <details> elements recursively.
+   * If found, scroll to element in viewport.
+   * @param {Element} element
+   */
+  _openClosestDetails(element) {
+    const details = element.closest('.lh-container details');
+    if (details instanceof HTMLElement) {
+      details.setAttribute('open', 'true');
+      if (details.parentElement) {
+        this._openClosestDetails(details.parentElement);
+      }
+      window.scroll({
+        top: details.offsetTop,
+        behavior: 'smooth',
+      });
+    } else {
+      throw new Error(`Closest ancestor '.lh-container details' of ${element.nodeName} not found`);
```

Comment:
Since this walks up recursively from the audit until it runs out of `<details>` parents, this will always eventually throw.

But I think we just don't need this at all...some audits may just not be nested in a `<details>` element (like the metrics (e.g. `#first-contentful-paint`)). No harm if it can't find any more closest `details`.